src/index.html

```diff
@@ -11,6 +11,40 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      data-qa="big"
+    >
+      <div>
+        <img
+          src="images/Search.svg"
+          alt="bigSerch"
+          class="buttonBig"
+        >
+        <input type="text"
```

Comment:
Don't forget to add an empty line between multiline siblings
```suggestion
        >
        
        <input type="text"
```